src/Artemis.UI/Styles/Artemis.axaml

```diff
@@ -6,4 +6,5 @@
     <!-- <FluentTheme Mode="Dark"></FluentTheme> -->
     <StyleInclude Source="avares://Material.Icons.Avalonia/App.xaml" />
     <StyleInclude Source="avares://Artemis.UI.Shared/Styles/Artemis.axaml" />
+    <StyleInclude Source="avares://AvaloniaEdit/AvaloniaEdit.xaml" />
```

Comment:
Please see if only adding it to the debug window also works, I'd rather not load another (potentially) big stylesheet on startup

Reply:
Yes, that works.